packages/salesforcedx-vscode-apex-debugger/.vscodeignore

```diff
@@ -6,3 +6,22 @@ src/**
 **/*.map
 .gitignore
 tsconfig.json
+
+# salesforcedx-apex-debugger files to ignore 
+node_modules/@salesforce/salesforcedx-apex-debugger/docs/**
+node_modules/@salesforce/salesforcedx-apex-debugger/src/**
+node_modules/@salesforce/salesforcedx-apex-debugger/test/**
+node_modules/@salesforce/salesforcedx-apex-debugger/out/test/**
+node_modules/@salesforce/salesforcedx-apex-debugger/package-lock.json
+node_modules/@salesforce/salesforcedx-apex-debugger/tsconfig.json
+
+# salesforcedx-utils-vscode files to ignore 
+node_modules/@salesforce/salesforcedx-apex-debugger/node_modules/@salesforce/salesforcedx-utils-vscode/node_modules/**
```

Comment:
Now that we create bundles for `salesforcefx-utils-vscode` we can remove its `node_modules`.